.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2.1
+
+jobs:
+  build:
+    machine:
+      image: ubuntu-2004:current
+    resource_class: medium
+    steps:
+      - run:
+          name: "Install build system dependencies"
+          command: |
+            sudo apt-get update
+            sudo apt-get install python3 python3-setuptools python3-pip
+            python3 -m pip install --user build
+      - checkout
+      - run:
+          name: "Build COMPASS python package"
+          command: |
+            python3 -m build --sdist --wheel .
```

Comment:
Thanks @rtburns-jpl. Compass depends on isce3 . The rest of dependencies are [here](https://github.com/opera-adt/COMPASS/blob/main/requirements.txt). I think if we add isce3 the rest of dependencies come with it. As we discussed we want to move forward assuming that compass works with isce3 on conda package.

Reply:
COMPASS also needs [s1-reader](https://github.com/opera-adt/s1-reader).

Reply:
@rtburns-jpl any update on this PR?

Reply:
Currently focused on getting https://github.com/opera-adt/COMPASS/pull/30 merged - once we do so, this PR can reuse the docker build scripts for the CI runs.

Reply:
#30 is merged, @rtburns-jpl you can go ahead and merge this PR if you are happy with it.